### PR TITLE
Documentation and configuration cleanup.

### PR DESCRIPTION
**Pull request by fluxsauce**
As requested in https://github.com/mafintosh/why-is-node-running/pull/29#issuecomment-367829655

### CODE REVIEW

**Comment by ralphtheninja** [comment]
LGTM except for the semicolons

File: package.json [hunk 1]
```diff
@@ -20,7 +23,13 @@
     "events",
     "handles"
   ],
-  "author": "Mathias Buus (@mafintosh)",
+  "author": "Mathias Buus <mathiasbuus@gmail.com>",
```
Comment (mafintosh) [inline package.json]: please do not include my email here

File: package.json [hunk 2]
```diff
@@ -1,14 +1,17 @@
 {
   "name": "why-is-node-running",
   "version": "2.0.2",
-  "description": "Node is running but you don't know why? why-is-node-running is here to help you.",
+  "description": "Determine why a node process won't exit",
```
Comment (mafintosh) [inline package.json]: i like the old desc :)

File: example.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-createServer()
-createServer()
+createServer();
+createServer();
 
 setTimeout(function () {
-  log()
-}, 100)
+    log();
+}, 100);
```
Comment (mafintosh) [inline example.js]: missing newline and some of the indent seems off above

**Comment by mafintosh** [comment]
Made some comments and also what @ralphtheninja said. Also I like prefer the old LICENSE file over the .md one. That's how I structure my things.

**Comment by fluxsauce** [comment]
All set. Might want to rebase instead of just merge if you're concerned about the history of `package.json`.

**Comment by mafintosh** [comment]
:+1:

**Comment by fluxsauce** [comment]
Thanks!

**Pull request merged**